lib/runPrettier.js

```diff
@@ -0,0 +1,70 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+/*
+ * Prettier must be ran from
+ * command line to make use of command line output.
+ * Use the bin file identified by Prettier's package.json
+ */
+function getPrettierBinLocation() {
```

Comment:
I've for whatever historical reason kept these kind of `just separating out code` functions as function declarations. No reason. Will change for conformity.